frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings/LookAndFeelSettings.tsx

```diff
@@ -208,18 +209,9 @@ export const LookAndFeelSettings = ({
       {!canWhitelabel && (
         <>
           <Divider my="2rem" />
-          <StaticEmbedSetupPaneSettingsContentSection
-            // eslint-disable-next-line no-literal-metabase-strings -- This only shows for admins
-            title={t`Removing the “Powered by Metabase” banner`}
-          >
-            {/* eslint-disable-next-line no-literal-metabase-strings -- This only shows for admins */}
-            <Text>{jt`This banner appears on all static embeds created with the Metabase open source version. You’ll need to upgrade to ${(
-              <ExternalLink
-                key="bannerPlan"
-                href={upgradePageUrl}
-              >{t`a paid plan`}</ExternalLink>
-            )} to remove the banner.`}</Text>
-          </StaticEmbedSetupPaneSettingsContentSection>
+          <div aria-label={t`Removing the banner`}>
```

Comment:
Adding a label since `StaticEmbedSetupPaneSettingsContentSection` also has a label for the section and it's used to target inside UI in some tests.

https://github.com/metabase/metabase/blob/7eaff412a3ad4cb509986f86bd340292647472b0/frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/common.unit.spec.tsx#L503-L507